Tools/PowershellModule/doc/new-winget-rest-source-azure.md

```diff
@@ -1,122 +1,118 @@
-# Windows Package Manager REST source
+# Create a REST source Windows Package Manager
 
-The Windows Package Manager provides a distribution channel for software packages containing their tools and applications. The instructions contained within this document provide guidance on how to setup a REST source that can be connected to using the Windows Package Manager. Providing a comprehensive package manager solution that consists of a command line tool and a set of services for installing applications on Windows 10.
+This section provides guidance on how to create a REST source that can be connected to using the Windows Package Manager for ISVs or Publishers who may have additional privacy requirements for their software distribution.
```

Comment:
This string is still confusingly phrased and off message guidance.

```suggestion
This section provides guidance on how to create a REST source for the Windows Package Manager. ISVs or Publishers may host and manage a rest source if they would like full control of the applications available in a source. An independently hosted source may choose to expose the read endpoints publicly or restrict access to specific IP address via the addition of a traffic shaping module.  The basic setups configured by the cmdlets and examples result in a source that is publicly readable but requires an authorization key to manage. 
```

Reply:
I don't think it's just ISVs or Publishers. It's essentially "anyone". I'd say "customers" here most likely.